client/sites-dashboard-v2/sites-dataviews/dataviews-fields/actions-field.tsx

```diff
@@ -1,13 +1,28 @@
 import { SiteActions } from '../sites-site-actions';
 import type { SiteExcerptData } from '@automattic/sites';
+import type { MouseEvent, KeyboardEvent } from 'react';
 
 type Props = {
 	site: SiteExcerptData;
 };
 
 const ActionsField = ( { site }: Props ) => {
+	const handleClickOrKeyDown = ( event: MouseEvent | KeyboardEvent ) => {
+		const target = event.target as HTMLElement;
+
+		// We have to bypass the link to prevent the page from full-reload
+		if ( ! target.closest( 'a' ) ) {
+			event.stopPropagation();
+		}
```

Comment:
This was confusing me as to why it was resolving the blocker with the hard reload.

If I understand correctly, if we stop propagation inside an anchor our router isn't able to recognize the event and do its work to navigate in the app smoothly. But stopping propagation inside the anchor prevents that from happening and defaults to standard anchor behavior of doing a full reload to the href?

Reply:
We have the [click event listener](https://github.com/Automattic/wp-calypso/blob/0fe02fe19479d37b5746a319069c3c165738199f/packages/calypso-router/src/index.js#L454) on the root to avoid the full reload.